pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -1016,29 +1014,27 @@ void Panels::MemoryCardListPanel_Simple::OnOpenItemContextMenu(wxListEvent& evt)
 
 	wxMenu* junk = new wxMenu();
 
-	if( idx != wxNOT_FOUND )
-	{
+	if (idx != wxNOT_FOUND) {
 		const McdSlotItem& card( GetCardForViewIndex(idx) );
 
-		if (card.IsPresent){
-			junk->Append( McdMenuId_AssignUnassign,	card.Slot>=0?_("Eject card"):_("Insert card ...") );
-			junk->Append( McdMenuId_Duplicate,	_("Duplicate card ...") );
-			junk->Append( McdMenuId_Rename,		_("Rename card ...") );
-			junk->Append( McdMenuId_Create,		_("Delete card") );
+		if (card.IsPresent) {
+			junk->Append( McdMenuId_AssignUnassign,	card.Slot>=0?_("Eject card"):_("&Insert card ...") );
+			junk->Append( McdMenuId_Duplicate,	_("D&uplicate card ...") );
```

Comment:
Hey, what happened here?

Reply:
Keyboard shortcuts.

Reply:
You can navigate via the options using the Alt key.

Reply:
They're shortcuts for the right-click menu. ;)

Reply:
I don't get it. _("D&uplicate card ...") ) is a keyboard shortcut? Could the ampersand be any place more random? :p

Reply:
There's another shortcut beginning with a d....

Reply:
I'll just file this under GUI coding adventures. Always something new and awesome to discover!

Reply:
The patch is fine by me, by the way. Good stuff :)

Reply:
Personally I suggest adding a new function for it to improve code readability. Something like ``MenuShortcut("Insert Card", "I")`` which returns the respective string with a shortcut letter of "I".

Any opinions? @FlatOutPS2 @turtleli

Reply:
> Personally I suggest adding a new function for it to improve code readability. Something like MenuShortcut("Insert Card", "I") which returns the respective string with a shortcut letter of "I".

I don't think that'd work well with localisation.

Reply:
Yeah, it won't. Bad idea :P